test/entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -e -u
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # start rsyslog
+rm -f /var/run/rsyslogd.pid
```

Comment:
Could you expand on the comment above the `rm` to explain briefly why we are trying to remove this pid file?

Given that we expect this `rm` to fail in most cases do you think it's also worth redirecting stderr to `/dev/null` so that nobody will confuse the `rm: cannot remove '/var/run/rsyslog.pid': No such file or directory` as an error to be investigated?